content/ja/docs/contribute/review/reviewing-prs.md

```diff
@@ -0,0 +1,85 @@
+---
+title: プルリクエストのレビュー
+content_type: concept
+main_menu: true
+weight: 10
+---
+
+<!-- overview -->
+
+ドキュメントのプルリクエストは誰でもレビューすることができます。Kubernetesのwebsiteリポジトリで[pull requests](https://github.com/kubernetes/website/pulls)のセクションに移動し、open状態のプルリクエストを確認してください。
+
+ドキュメントのプルリクエストのレビューは、Kubernetesコミュニティに自分を知ってもらうためのよい方法の1つです。コードベースについて学んだり、他のコントリビューターとの信頼関係を築く助けともなるはずです。
+
+レビューを行う前には、以下のことを理解しておくとよいでしょう。
+
+- [コンテンツガイド](/docs/contribute/style/content-guide/)と[スタイルガイド](/docs/contribute/style/style-guide/)を読んで、有益なコメントを残せるようにする。
+- Kubernetesのドキュメントコミュニティにおける[役割と責任](/docs/contribute/participate/roles-and-responsibilities/)の違いを理解する。
+
+<!-- body -->
+
+## はじめる前に
+
+レビューを始める前に、以下のことを心に留めてください。
+
+- [CNCFの行動規範](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)を読み、いかなる時にも行動規範にしたがって行動するようにする。
+- 礼儀正しく、思いやりを持ち、助け合う気持ちを持つ。
+- 変更点だけでなく、PRのポジティブな側面についてもコメントする。
```

Comment:
By comparing https://github.com/kubernetes/website/blob/master/content/en/docs/contribute/review/reviewing-prs.md the following line seems missing:
```
Be empathetic and mindful of how your review may be received.
```

Reply:
@oomichi Thank you for your careful review! It's my mistake. I updated the translation to include this sentence too.